src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -65,6 +65,11 @@ internal class ConsoleLogger : ITestLoggerWithParameters
         /// </summary>
         public const string ExtensionUri = "logger://Microsoft/TestPlatform/ConsoleLogger/v1";
 
+        /// <summary>
+        /// Represents failed tests. 
+        /// </summary>
+        private const string Failed = "Failed";
```

Comment:
How will this get localized ?
if you don't want them localized, why dint you use the TestOutcome.Failed instead ?